### PR TITLE
fix: fix payload of create api user

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [x] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
- edit parameter of user create API

### CODE REVIEW

**Review by sulmoJ — APPROVED** [review]
LGTM!! (with. @ImMin5 )

**Review by ImMin5 — APPROVED** [review]
LGTM

File: apps/web/src/services/administration/iam/user/modules/user-management-modal/UserManagementModal.vue [hunk 1]
```diff
@@ -205,7 +205,14 @@ export default {
...
             if (userPageState.visibleCreateModal) {
-                data.backend = formState.activeTab === 'local' ? 'LOCAL' : 'EXTERNAL';
+                if (formState.activeTab === 'local' || formState.activeTab === 'apiOnly') {
+                    data.backend = 'LOCAL';
+                    if (formState.activeTab === 'apiOnly') {
+                        data.user_type = 'API_USER';
+                    }
+                } else {
+                    data.backend = 'EXTERNAL';
+                }
```
Comment (WANZARGEN) [inline apps/web/src/services/administration/iam/user/modules/user-management-modal/UserManagementModal.vue:215]: nitpick: I think we can make it more explicit. :)

```suggestion
                if (formState.activeTab === 'local') {
                    data.backend = 'LOCAL';
                }  else if (formState.activeTab === 'apiOnly') {
                        data.backend = 'LOCAL';
                        data.user_type = 'API_USER';
                } else {
                    data.backend = 'EXTERNAL';
                }
```

**Review by WANZARGEN — APPROVED** [review]
I left a nitpick comment!

**Pull request merged**